test/other/metadce/test_metadce_hello_O0.gzsize

```diff
@@ -1 +1 @@
-8137
```

Comment:
what codegen change is caused by the version update that changes the output size?

Reply:
```
$ diff -u out/test/a.out.js  new/a.out.js 
--- out/test/a.out.js	2024-05-28 15:36:05.651698784 -0700
+++ new/a.out.js	2024-05-28 15:36:10.411694378 -0700
@@ -16,8 +16,8 @@
   }
   var ca = process.versions.node, t = ca.split(".").slice(0, 3);
   t = 10000 * t[0] + 100 * t[1] + 1 * t[2].split("-")[0];
-  if (160000 > t) {
-    throw Error("This emscripten-generated code requires node v16.0.0 (detected v" + ca + ")");
+  if (182003 > t) {
+    throw Error("This emscripten-generated code requires node v18.20.3 (detected v" + ca + ")");
   }
   var fs = require("fs"), da = require("path");
   n = __dirname + "/";
```